pyproject.toml

```diff
@@ -14,7 +14,7 @@ repository = "https://github.com/canonical/postgresql-test-app"
 [tool.poetry.dependencies]
 python = "^3.10"
 ops-lib-pgsql = "^1.4"
-ops = "^2.16.1"
+ops = "^2.17.0"
```

Comment:
it has been updated already new in PR #225: https://github.com/canonical/postgresql-test-app/pull/225/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R17
@dragomirp can you explain it?

Reply:
The commits in main are empty: https://github.com/canonical/postgresql-test-app/commit/6637b6bd2d6ae73441e8de74726331ec77dd5207
as the content is already merged in PR #223.

Q: is renovate went crazy to propose the same changes again? CC: @carlcsaposs-canonical JFYI.